src/modules/common/components/pageLoader/PageLoader.js

```diff
@@ -15,6 +15,11 @@ function PageLoader() {
           ))}
         </div>
       </figure>
+      {progress && (
+        <div>
+          Loading <span>{progress}%</span>
```

Comment:
Not my forte but just sharing an feedback from users perspective.

It is a bit confusing as an user what the progress percentage refers to, is it the app getting it's views ready, or waiting for some meta information from server, or the server is getting ready to serve the app.  It could be more clear if we mention something which points to the fact that blockchain indexing is in progress. Ex: `Indexing Blockchain. 50% Done..`

Reply:
@ManuGowda @Ade-kola what's your thought on this.